mobile-ui/src/main/java/org/sagebionetworks/research/mobile_ui/show_step/view/FragmentNavigationRule.java

```diff
@@ -0,0 +1,5 @@
+package org.sagebionetworks.research.mobile_ui.show_step.view;
+
+public interface FragmentNavigationRule {
```

Comment:
What are the FragmentRules for?

Reply:
Oops those were used for the hand selection for a while but I decided it was better to make all that functionality possible from a step. I'll remove these.